### PR TITLE
Remove Compose file v1 ref docs

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

- Remove Compose file version 1 docs as this has been deprecated
- Added a redirect to point to Compose spec topic

### CODE REVIEW

File: compose/compose-file/compose-file-v2.md [hunk 1]
```diff
@@ -998,7 +998,7 @@ options:
 
 > Changed in [version 2](compose-versioning.md#version-2) file format.
 >
-> The `network_mode` option replaces the version 1 [net](compose-file-v1.md#net) option.
+> The `network_mode` option replaces the Compose file version 1 net option.
```
Comment (chris-crone) [inline compose/compose-file/compose-file-v2.md:1001]: Should we remove this or add `(deprecated)` so that people aren't confused about v1 here?

**Review by chris-crone — APPROVED** [review]
One question but otherwise LGTM!

File: compose/compose-file/compose-file-v2.md (hunk 1 above)
Comment (usha-mandya) [reply compose/compose-file/compose-file-v2.md:1001]: You are right. This piece of info doesn't add value to users. I think it's best to remove. Let me update the PR.

**Review by chris-crone — APPROVED** [review]
LGTM

**Pull request merged**